vertical-pod-autoscaler/pkg/recommender/routines/recommender.go

```diff
@@ -160,7 +153,7 @@ func (r *recommender) RunOnce() {
 
 	r.clusterStateFeeder.LoadRealTimeMetrics()
 	timer.ObserveStep("LoadMetrics")
-	klog.V(3).Infof("ClusterState is tracking %v PodStates and %v VPAs", len(r.clusterState.Pods), len(r.clusterState.Vpas))
+	klog.V(3).InfoS("ClusterState is tracking", "Pods", len(r.clusterState.Pods), "VPAs", len(r.clusterState.Vpas))
```

Comment:
nit: consistency. Let's get the casing consistent in the lines we  change. Most others are lowercase.